lib/signer.rb

```diff
@@ -14,8 +14,8 @@ class Signer
   WSU_NAMESPACE = 'http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-wssecurity-utility-1.0.xsd'
   WSSE_NAMESPACE = 'http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-wssecurity-secext-1.0.xsd'
 
-  def initialize(document)
-    self.document = Nokogiri::XML(document.to_s, &:noblanks)
+  def initialize(document, should_trim: true)
```

Comment:
I think it would be better to stick with nokogiri naming:

```
def initialize(document, noblanks: true)
  self.document = Nokogiri::XML(document.to_s) do |config|
     config.noblanks if noblanks
  end
  ...
```